subworkflows/nf-core/abundance_differential_filter/tests/main.nf.test.snap

```diff
@@ -0,0 +1,352 @@
+{
+    "limma - basic - microarray": {
+        "content": [
+            [
+                
+            ],
+            [
+                
```

Comment:
missing elements here....expected ?

Reply:
Dammit- good point, I'll figure this out